### ISSUE TITLE
DCS World Scripting Engine definitions doc to separate repository?

### ISSUE THREAD

**Issue by birgersp** [FlightControl-Master/MOOSE#147]
Hi,

The documentation files located in `Moose Development/Dcs/` could be used outside of MOOSE. Since they describe the definitions already provided in DCS, this documentation is very useful even if you're not going to use MOOSE. So people might want to copy it without the rest of MOOSE.

I suggest this documentation could be put in a separate repository, and added as a submodule to this repository. Internally (for MOOSE), the location  of the files could be excactly as it is now.

I'm copying this documentation for now (since I need it for a mission script), but it would be nice to not maintain this documentation two separate places.

TL;DR:
1. Move the contents of `Moose Development/Dcs` to a spearate repo
2. Add the separate repo as a submodule in `Moose Development/Dcs`

**Comment by FlightControl-Master** [comment]
Birger, I saw a pull request. Why a pull request? It confuses me. Can you explain a bit how this works?

**Comment by birgersp** [comment]
([Discussion in #148](https://github.com/FlightControl-Master/MOOSE/pull/148#issuecomment-267920731))

**Comment by birgersp** [comment]
(Referring to our Slack discussion), I suggest that this pull request is reverted (reset master to before these commits). Some of the files that were to be moved are modified since this pull request was issued, and I'm afraid the files are now a bit mixed. When the PR is reverted, and if you're still willing to separate the docs I'll take care of the following:

- Update the luadoc repo so it matches your changes in MOOSE
- Transfer luadoc repo to you
- Remove files from MOOSE
- Add luadoc repo as submodule to MOOSE
- Make a new pull request
- (Everybody needs to stop developing the luadoc in MOOSE until the PR is merged)
